tools/checkcompletion.sh

```diff
@@ -127,7 +127,7 @@ check_completion_file_integrity() {
 	fi
 
 	comp_opts="$( cat "${completion_dir}/${prog}" \
-			| grep -o -P '[[:space:]]*--(?![^[:alnum:]_])[A-Za-z-_.0-9]*' \
+			| grep -o -P '[[:space:]]*--(?![^[:alnum:]])[A-Za-z-.0-9]*' \
```

Comment:
Deleting both `_`, an option having `_` in the middle of its name is also ignored.
From the point of view of the lsfd command, just ignoring the option started from `--_` is enough.

```
$ echo --example |  grep -o -P '[[:space:]]*--(?![^[:alnum:]])[A-Za-z-.0-9]*'
--example
$ echo --example_opt |  grep -o -P '[[:space:]]*--(?![^[:alnum:]])[A-Za-z-.0-9]*'
--example
$ echo --example_opt |  grep -o -P '[[:space:]]*--(?![^[:alnum:]])[A-Za-z-.0-9_]*'
--example_opt
```